### PR TITLE
Fix typo in features.html

### PR DESCRIPTION
**Pull request by sappharx**

### CODE REVIEW

File: features.html [hunk 1]
```diff
@@ -213,12 +213,12 @@ <h2 id="_product_types">Product Types</h2>
...
 <pre class="prettyprint highlight"><code class="language-java" data-lang="java">// Regular Java
-public Integer albuquerqueToLA(Map&gt; map) {
+public Integer albuquerqueToLA(Map&lt;String, Integer&gt; map) {
   Map m = map.get("Albuquerque");
   if (m != null) return m.get("Los Angeles"); // May return null.
 }
 // Functional Java with product and option types.
-public Option albuquerqueToLA(TreeMap, Integer&gt;() map) {
+public Option albuquerqueToLA(TreeMap&lt;String, Integer&gt;() map) {
   return m.get(p("Albuquerque", "Los Angeles"));
```
Comment (sappharx) [inline features.html]: should `m.get` be `map.get`?

**Comment by gliptak** [comment]
@sappharx Thank you. For the first typo the update needs to happen at https://github.com/functionaljava/functionaljava.github.io/blob/gradle/src/jbake/content/features.adoc (note branch ...) then the pages generated using the process described in https://github.com/functionaljava/functionaljava.github.io/blob/master/README.adoc

**Comment by sappharx** [comment]
Thanks for the heads up; I'll fix that when I've got a little more time this evening

**Comment by gliptak** [comment]
Updated with https://github.com/functionaljava/functionaljava.github.io/pull/7

Thanks for pointing this out @sappharx

**Comment by sappharx** [comment]
Sorry for not finishing this off earlier, but the `./gradlew publish` kinda scared me off a bit; plus it looks like I didn't quite get it right, either

**Comment by gliptak** [comment]
@sappharx I think only only project members are able to ```publish```, but you can submit PRs to update the ```adoc``` files on the ```gradle``` branch.

**Comment by sappharx** [comment]
ok, I just remembered that I couldn't get the `jBakeServe` command to actually serve anything, either

**Comment by gliptak** [comment]
You have ```jbake``` to (re)generate content, and run ```jbakeServer``` to serve content to your browser.

**Pull request closed without merging**